content/docs/studio/visualize-exp.md

```diff
@@ -0,0 +1,42 @@
+# Visualize and compare experiements
```

Comment:
```suggestion
# Visualize and Compare Experiments
```

Reply:
Should "compare" be reflected in the sidebar label?

Reply:
Yeah I think this whole title can be changed since it´s a bit repetitive to speak of Views in the first 2 pages and then of Visualizing in this one. Let me read the contents to come up with specific suggestions ⌛

Reply:
please, let's not do this `Yeah I think this whole title can be changed` - we don't have time for this. Let's move it as a checkbox somewhere. Fix low hanging fruits, obvious bugs, etc.

Reply:
> we don't have time for this... Fix low hanging fruits

Yeah my initial review only focused on the overall structure with that same idea. And before reading all the content I was pretty confused by the titles. Isn't a "View" a visualization?

In fact reading into the View page, it's defined as  "an interactive representation of the experiments run in your ML project," which sounds the same as visualizing experiments (fine if we plan to change the term "View" later though).

After reading the content in this page in detail I see that we mean specific experiments, which makes total sense. But before that context, this was confusing at first glance.

But yes I see it's not low-hanging fruit so let's just leave this unresolved for now and we'll revisit later. Thanks

Reply:
Added to new ticket https://github.com/iterative/dvc.org/issues/2468